wdl/tasks/illumina.wdl

```diff
@@ -0,0 +1,46 @@
+version 1.0
+
+
+import "../structs/resources.wdl"
+import "../structs/fastq.wdl"
+
+
+task trim_adapters {
+    input {
+        FastqPair fastqs
+        File adapters
+        Resources resources
+        String adapter1 = "P5"
+        String adapter2 = "P7"
+        String read1_out_filename = "trim.R1.fastq.gz"
+        String read2_out_filename = "trim.R2.fastq.gz"
+        String trimstats_out_filename = "trimstats.txt"
+    }
+
+    command {
+        trim-adapters-illumina \
+            -f ~{adapters} \
+            -1 ~{adapter1} \
+            -2 ~{adapter2} \
+            --threads=~{resources.cpu} \
```

Comment:
You didn't want to wrap these in TrimAdapterParams? Also here I let threads be independently specified even though is should probably be the same as cpu number: https://github.com/ENCODE-DCC/dnase-seq-pipeline/blob/363a4eca990d310ed3e5c2bc72b20302b60442a7/wdl/tasks/bwa.wdl#L55 Should we pick one way or the other?

Reply:
Case by case basis agreed.